frappe/email/doctype/auto_email_report/auto_email_report.py

```diff
@@ -169,15 +173,21 @@ def send_now(name):
 
 def send_daily():
 	'''Check reports to be sent daily'''
-	now = frappe.utils.now_datetime()
-	for report in frappe.get_all('Auto Email Report',
-		{'enabled': 1, 'frequency': ('in', ('Daily', 'Weekly'))}):
+
+	current_day = calendar.day_name[now_datetime().weekday()]
+	enabled_reports = frappe.get_all('Auto Email Report',
```

Comment:
indenting !

Reply:
Sorry about that. I've fixed the indents.